samples/bluetooth/shell_bt_nus/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
+  description: BT NUS shell sample
+  name: BT NUS shell
+tests:
+  test_build:
+    build_only: true
+    build_on_all: true
+    platform_whitelist: nrf51_pca10028 nrf52_pca10040 nrf52840_pca10056 nrf52810_pca10040
```

Comment:
what about nrf91?

Reply:
It does not have Bluetooth :)

Reply:
good point :D

Reply:
first we need to implement bluetooth stack on that radio. next sprint? 😄